docs/source/serving/update.rst

```diff
@@ -0,0 +1,97 @@
+.. _update:
+
+Update
+===========
+
+SkyServe supports rolling update for your services. Use ``sky serve update`` to update an existing service:
+
+.. code-block:: console
+
+    $ sky serve update service-name new_service.yaml
+
+SkyServe will launch new replicas described by ``new_service.yaml``. When the number of new replicas reaches the minimum number of replicas (``min_replicas``) required for the service, SkyServe will scale down old replicas to save cost. SkyServe allows users to update ``replica_policy`` parameters, such as ``target_qps_per_replica``. SkyServe also allows users to update ``resources`` parameters, such as ``cpu`` and ``memory``, so that new replicas can be launched on VMs of different types.  
```

Comment:
Add a sentence to say, only updating target_qps will still restart all your replicas? this could be unexpected.

Reply:
Also, say the endpoint will remain unchanged through the process? And it should always be accesible and no interruption will incur

Reply:
Maybe we can leave out these cases in the documentation -- as ideally we want to fix them in the future.

Reply:
makes sense